docs/github-issue-guide.md

```diff
@@ -1,87 +0,0 @@
-# How to file a GitHub Issue
-
-We use GitHub Issues to log all of our todos and tasks. Here is
-[a good guide](https://guides.github.com/features/issues/) for them if you are
-unfamiliar.
-
-When logging an issue with go-ipfs, it would be useful if you specified the
-below information, if possible. This will help us triage the issues faster.
-Please title your issues with the type. For instance:
-
-- "bug: Cannot add file with `ipfs add`"
-- "question: How do I use `ipfs block <hash>`?"
-
-Putting the command in backticks helps us parse the natural language description,
-and is generally suggested.
-
-This is a _living guide_. If you see anything that should be here and isn't, or
-have ideas on improvement, please open a "meta" issue.
-
-### Type
-
-- "bug": If what you are filing is a bug.
-- "meta": If it is something about how we run go-ipfs, and not code related in itself.
-- "question": If you have a question.
-- "test failure": If the tests are failing
-- "panic": If it is a severe bug.
-- "enhancement ": If you have a feature you would like that enhances go-ipfs.
-
-### Platform
-
-For platform and processor, just run `ipfs version --all` and include that output.
-
-Your platform.
-
-- "Linux"
-- "Windows"
-- "Mac"
-- Etc.
-
-### Processor
-
-Your processor architecture.
-
-- "x86"
-- "amd64"
-- "Arm"
-
-### Area
-
-What your issue refers to. Multiple items are OK.
```

Comment:
This didn't exist on the issue template. Should we put it in? Most users probably won't know enough to fill this out, unfortunately. We usually just use labels.

Reply:
It could be useful, but you are right most users won't know enough to fill this out.  Users who do likely already have write access and can assign lables, that is if they remember to do so. :)

Reply:
That *never* happens. I have no idea what you're talking about.

Reply:
I used to tag issues with subsystem/topic labels. It was endless work and it somehow stopped being important.